app/models/test_track/session.rb

```diff
@@ -30,18 +30,12 @@ def state_hash
     }
   end
 
-  def log_in!(*args) # rubocop:disable Metrics/AbcSize, Metrics/MethodLength
+  def log_in!(*args)
     opts = args[-1].is_a?(Hash) ? args.pop : {}
 
-    if args[0].is_a?(TestTrack::Identity)
-      identity = args[0]
-      identifier_type = identity.test_track_identifier_type
-      identifier_value = identity.test_track_identifier_value
-    else
-      identifier_type = args[0]
-      identifier_value = args[1]
-      warn "#log_in! with two args is deprecated. Please provide a TestTrack::Identity"
-    end
+    identity = args[0]
+    identifier_type = identity.test_track_identifier_type
+    identifier_value = identity.test_track_identifier_value
```

Comment:
I think we no longer use opts, and we can just name the args individually instead of taing *args, right?

Reply:
oh we use opts. Are we ruby 2-only yet? We could use a named optional arg instead of opts too.

Reply:
yup we're `>= 2.1.0`